pkgs/development/tools/refurb/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "refurb";
+  version = "1.5.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "dosisod";
+    repo = "refurb";
+    rev = "v${version}";
+    hash = "sha256-bWhIt91wGCOlknSVd12OgCKsPUw8iS3OSiKnsspxzPI";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    mypy
+    mypy-extensions
+    tomli
+    typing-extensions
+  ];
+
+  checkInputs = with python3Packages; [
+    pytest
+    attrs
+    black
+    click
+    colorama
+    coverage
+    flake8
+    iniconfig
+    isort
+    mccabe
+    packaging
+    pathspec
+    platformdirs
+    pluggy
+    py
+    pycodestyle
+    pyflakes
+    pyparsing
+    pytest
+    pytest-cov
```

Comment:
```suggestion
```
pytest is included in pytestCheckHOok and and we do not want to run cov with pytest-cov because it has no value for us and could even fail over time if upstream defines a minimal coverage.